### ISSUE TITLE
No class def error

### ISSUE THREAD

**Issue by Ghost-chu** [SkytAsul/BeautyQuests#24]
**Describe the bug**
Plugin BeautyQuests v0.12.2 generated an exception while executing task 11509758
java.lang.NoClassDefFoundError: com/sk89q/worldedit/Vector
**To Reproduce**
1.Create a find a region quest.
2,Walk argound.
3,Look console.
**Expected behavior**
When i create a find a region quest, plugin will throw exception,
**Screenshots, errors**
Is there stacktraces in the console ?

```
[13:24:58 WARN]: [fr.skytasul.quests.BeautyQuests] [BeautyQuests] Plugin BeautyQuests v0.12.2 generated an exception while executing task 11509758
java.lang.NoClassDefFoundError: com/sk89q/worldedit/Vector
at fr.skytasul.quests.utils.compatibility.WorldGuard.isInRegion(WorldGuard.java:73) ~[?:?]
at fr.skytasul.quests.stages.StageArea$1.run(StageArea.java:46) ~[?:?]
at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftTask.run(CraftTask.java:63) ~[PaperSpigot-1.13.2-R0.1.jar:git-Paper-459]
at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52) ~[PaperSpigot-1.13.2-R0.1.jar:git-Paper-459]
at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[PaperSpigot-1.13.2-R0.1.jar:git-Paper-459]
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_192]
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_192]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_192]
```

**Softwares (please complete the following information):**
BeautyQuests version (X.X.X, not "latest"), Game version, Server version
Are you using a protocol hack (like ViaVersion) ?
0.12.2 1.13.2 1.13.2 no

**Additional context**
Lastet WorldGuard, Lastet WorldEdit
(FR: Si vous êtes plus facilement compréhensible en français, écrivez dans cette langue car il s'agit de ma langue natale)

**Comment by SkytAsul** [comment]
Problem with WorldEdit's API who has been edited, will be fixed in 0.13

**Comment by SkytAsul** [comment]
Fixed in latest release
